testbed/tf-script/xpanse-ui.tf

```diff
@@ -0,0 +1,21 @@
+resource "clevercloud_docker" "xpanse_ui" {
+  name               = "ui"
+  region             = "par"
+  min_instance_count = 1
+  max_instance_count = 1
+  smallest_flavor    = "M"
+  biggest_flavor     = "M"
+
+  deployment {
+    repository = var.docker_repository_url
+    commit     = var.docker_commit
+  }
+
+  dockerfile = "testbed/clever-cloud/xpanse-ui/Dockerfile"
+
+  environment = {
+    CC_DOCKER_EXPOSED_HTTP_PORT = "3000"
```

Comment:
we are missing one variable. 
https://github.com/eclipse-xpanse/xpanse-ui/blob/main/.env#L1
This must be the xpanse API URL. 
Please also once test if the UI is really sending reuqests to the correct xpanse-app which you have deployed.